pinot-common/src/main/java/org/apache/pinot/common/function/AggregationFunctionType.java

```diff
@@ -79,7 +79,7 @@ public boolean isOfType(AggregationFunctionType... aggregationFunctionTypes) {
    * Returns the corresponding aggregation function type for the given function name.
    */
   public static AggregationFunctionType getAggregationFunctionType(String functionName) {
-    String upperCaseFunctionName = functionName.toUpperCase();
+    String upperCaseFunctionName = functionName.toUpperCase().replace("_", "");
```

Comment:
This will break `ST_Union`. Can we first look up with original function name, and if not found, replace `_` then?

Reply:
will do that